packages/frpc/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE=https://gofrp.org/doc
+TERMUX_PKG_DESCRIPTION="An offline tool for querying IP geographic information and CDN provider."
```

Comment:
Is this description correct? https://github.com/fatedier/frp says:
```
A fast reverse proxy to help you expose a local server behind a NAT or firewall to the internet.
```

Reply:
my mistake, will fix it.